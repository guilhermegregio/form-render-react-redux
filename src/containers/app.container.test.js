import React from 'react';
import ReactDOM from 'react-dom';
import { Provider } from 'react-redux';

import AppContainer from './app.container';
import configureStore from '../store/store';

import RequestFields from '../mock.json';

const store = configureStore({formRender: RequestFields.fields});

it('default render app container', () => {
  const div = document.createElement('div');
  ReactDOM.render(<Provider store={store}><AppContainer /></Provider>, div);

  expect(div).toMatchSnapshot();
});
