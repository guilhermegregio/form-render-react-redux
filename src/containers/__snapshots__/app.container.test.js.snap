// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`default render app container 1`] = `
<div>
  <div
    data-reactroot=""
  >
    <div>
      <div
        class="text-component group undefined"
      >
        <input
          type="text"
          id="txtFullname"
          name="txtFullname"
          placeholder=""
          required=""
          value=""
        />
        <span
          class="highlight"
        />
        <span
          class="bar"
        />
        <label>
          <!-- react-text: 8 -->
          Nome
          <!-- /react-text -->
          <!-- react-text: 9 -->
          :
          <!-- /react-text -->
        </label>
      </div>
      <div
        class="text-component group undefined"
      >
        <input
          type="text"
          id="txtCpf"
          name="txtCpf"
          placeholder=""
          required=""
          value=""
        />
        <span
          class="highlight"
        />
        <span
          class="bar"
        />
        <label>
          <!-- react-text: 15 -->
          CPF
          <!-- /react-text -->
          <!-- react-text: 16 -->
          :
          <!-- /react-text -->
        </label>
      </div>
      <div
        class="text-component group undefined"
      >
        <input
          type="text"
          id="txtTelefone"
          name="txtTelephone"
          placeholder=""
          required=""
          value=""
        />
        <span
          class="highlight"
        />
        <span
          class="bar"
        />
        <label>
          <!-- react-text: 22 -->
          Telefone
          <!-- /react-text -->
          <!-- react-text: 23 -->
          :
          <!-- /react-text -->
        </label>
      </div>
      <div
        class="text-component group undefined"
      >
        <input
          type="text"
          id="txtAddress"
          name="txtAddress"
          placeholder=""
          required=""
          value=""
        />
        <span
          class="highlight"
        />
        <span
          class="bar"
        />
        <label>
          <!-- react-text: 29 -->
          Endere√ßo
          <!-- /react-text -->
          <!-- react-text: 30 -->
          :
          <!-- /react-text -->
        </label>
      </div>
      <div
        class="map-component group undefined"
      />
      <div
        class="text-component group undefined"
      >
        <input
          type="text"
          id="txtComplement"
          name="txtComplement"
          placeholder=""
          required=""
          value=""
        />
        <span
          class="highlight"
        />
        <span
          class="bar"
        />
        <label>
          <!-- react-text: 37 -->
          Complemento
          <!-- /react-text -->
          <!-- react-text: 38 -->
          :
          <!-- /react-text -->
        </label>
      </div>
      <div
        class="upload-component group undefined"
      >
        <input
          type="file"
          id="uplImage"
          name="uplImage"
        />
      </div>
      <div
        class="image-component group undefined"
      >
        <img
          name="imgAvatar"
          alt="imgAvatar"
        />
      </div>
      <div
        class="button-component group undefined"
      >
        <button
          id="btnSave"
          type="button"
          class="button primary"
          value="Gravar"
        >
          <span
            class="load fa fa-spinner"
          />
          <span
            class="value"
          >
            Gravar
          </span>
        </button>
      </div>
    </div>
  </div>
</div>
`;
